k6bench.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# External dependencies:
+# - https://www.selenic.com/smem/
+# - https://stedolan.github.io/jq/
+set -eEuo pipefail
+
+trap 'cleanup; exit 0' EXIT
+trap 'trap - INT; cleanup; kill -INT $$' INT
+trap 'trap - TERM; cleanup; kill -TERM $$' TERM
+
+# Create temp files to store the output of each collection command.
+cpuf=$(mktemp)
+memf=$(mktemp)
+vusf=$(mktemp)
+rpsf=$(mktemp)
+
+cleanup() {
+  rm -f "$cpuf" "$memf" "$vusf" "$rpsf"
```

Comment:
In general I prefer to wrap variables in quotes and in `${}` not only prefix them with `$`. 

IIRC that also prevents you from potentially matching a shorter variable that is just prefix of the longer one.

Reply:
Right, I do that as well where it's useful (e.g. part of an interpolated string), but in this case there's no reason to do so, since they're separate arguments, and there's no risk of mistakenly using a different variable.